src/services/ipfsService.ts

```diff
@@ -92,41 +91,18 @@ export default class IPFS {
       });
     }
     try {
-      const uploadedDirectory = [];
       const files = sources.map((source, index) => {
         return {
           path: isErc1155 ? `${index}.json` : `${index}`,
           content: source,
         };
-      }) as ImportCandidateStream;
-
-      for await (const file of this.ipfsClient.addAll(files, {
-        wrapWithDirectory: true,
-      })) {
-        uploadedDirectory.push(file);
-      }
-      const item = [...uploadedDirectory].pop();
-      return `ipfs://${item?.cid.toString()}/`;
-    } catch (error) {
-      return log.throwError(Logger.message.an_error_occured_with_ipfs_api, Logger.code.IPFS, {
-        location: Logger.location.IPFSSERVICE_UPLOADDIRECTORY,
-        error,
       });
-    }
-  }
-
-  async unPinFile({ hash }: { hash: string }) {
```

Comment:
It looks like this `unPinFile` function was removed.  Does it need to stay?

Reply:
we don't expose it through the main file ( index.js ) , so even with the old version this function wasn't available for users